src/components/Avatar/Avatar.tsx

```diff
@@ -0,0 +1,54 @@
+import { useMemo } from 'react';
+
+import {
+  AvatarContainer,
+  AvatarProps,
+  AvatarWrapper,
+  EditButtonWrapper,
+} from './Avatar.style';
+import EditButton from './EditButton';
+
+/**
+ * @param [size='S'] - XS, S, M, L, XL
+ * @param [shape='circle'] - circle은 원형, square는 사각형, round는 둥근 사각형
+ * @param src - 이미지 소스
+ * @param [isBorder=true] - true는 보더(solid) 적용
+ * @param [isShadow=true] - true는 그림자 적용
+ * @param [isEdit=false] - true는 편집 아이콘 적용
+ * @param onClick - 마우스 클릭 핸들러 함수
+ */
+export default function Avatar({
+  size = 'S',
+  shape = 'circle',
+  src,
+  isBorder = true,
+  isShadow = true,
+  isEdit = false,
+  onClick,
+  ...props
+}: AvatarProps) {
+  const isPointer = useMemo(() => {
+    return onClick ? true : false;
+  }, []);
+
+  return (
+    <AvatarContainer
+      $isPointer={isPointer}
+      onClick={onClick}
+    >
+      <AvatarWrapper
+        size={size}
+        shape={shape}
+        src={src}
+        $isBorder={isBorder}
+        $isShadow={isShadow}
+        {...props}
+      ></AvatarWrapper>
```

Comment:
사소한 부분이지만 children이 없는 경우는 바로 tag를 닫아주는건 어떨까요?!

```suggestion
      <AvatarWrapper
        size={size}
        shape={shape}
        src={src}
        $isBorder={isBorder}
        $isShadow={isShadow}
        {...props}
      />
```

Reply:
이거하고 관련해서 단일 태그는 자동으로 하나로 닫히도록 eslint 설정이 있던데 따로 설정을 추가하지 않아도 될까요?